README.md

```diff
@@ -1,5 +1,7 @@
 [Learn about Nim][nim]
 
+[![Build Status](https://travis-ci.org/hacksaurz/nim.svg?branch=master)](https://travis-ci.org/hacksaurz/nim)
```

Comment:
See it live [here](https://github.com/hacksaurz/nim/tree/NIM-9-Travis). Note that it's failing because `master` doesn't have `.travis.yml`, so let's merge this PR! :smile:

Reply:
OMG I was just looking at that and was coming back here to comment. It updates live in the readme?! Are you a fucking wizard?!